### PR TITLE
store/tikv: retry TSO RPC

### PR DESCRIPTION
**Pull request by youjiali1995**
Signed-off-by: youjiali1995 <zlwgx1023@gmail.com>

### What problem does this PR solve?

Problem Summary:

Should use `getTimestampWithRetry` instead of `GetTimestamp`.

### What is changed and how it works?

What's Changed:

Replace all `getTimestampWithRetry` by `GetTimestamp`.

How it Works:

### Related changes

- Need to cherry-pick to the release branch

### Check List 

Tests 

- No code

### Release note 

- No release note.

### CODE REVIEW

File: store/tikv/lock_resolver.go [hunk 1]
```diff
@@ -243,7 +238,7 @@ func (lr *LockResolver) BatchResolveLocks(bo *Backoffer, locks []*Lock, loc Regi
 		metrics.LockResolverCountWithExpired.Inc()
 
 		// Use currentTS = math.MaxUint64 means rollback the txn, no matter the lock is expired or not!
-		status, err := lr.getTxnStatus(bo, l.TxnID, l.Primary, callerStartTS, math.MaxUint64, true, false, l)
+		status, err := lr.getTxnStatus(bo, l.TxnID, l.Primary, math.MaxUint64, math.MaxUint64, true, false, l)
```
Comment (sticnarf) [inline store/tikv/lock_resolver.go:241]: It seems a bit more reasonable to me to set 0 to `callerStartTS` when it is actually meaningless.

**Review by sticnarf — COMMENTED** [review]
The rest lgtm.

File: store/tikv/lock_resolver.go (hunk 1 above)
Comment (cfzjywxk) [reply store/tikv/lock_resolver.go:241]: We may add some new comments for the `getTxnStatus` as it's a bit complex to use now.

**Review by sticnarf — APPROVED** [review]

**Review by cfzjywxk — APPROVED** [review]
/lgtm

**Pull request merged**